src/utils/localeDataRegistry.js

```diff
@@ -0,0 +1,10 @@
+import IntlMessageFormat from 'intl-messageformat'
+
+export default function (data = []) {
+  const locales = Array.isArray(data) ? data : [data]
+
+  locales.forEach((localeData) => {
```

Comment:
Are you sure about the name `locales` ? Because `locales` have ... `localeData`. `localeData` have ... `locale`
This is weird